Index: views_data_export_phpexcel.module
<+>UTF-8
===================================================================
--- views_data_export_phpexcel.module	(date 1414763539000)
+++ views_data_export_phpexcel.module	(date 1543601672000)
@@ -17,8 +17,8 @@
     'version arguments' => array(
       'file' => 'changelog.txt',
       // 1.8.x: Version 1.8.0
-      'pattern' => '/@version\s+(\d+\.\d+\.\d+)/',
-      'lines' => 25,
+      'pattern' => '/(v\d+\.\d+\.\d+)/',
+      'lines' => 26,
     ),
     'files' => array(
       'php' => array('PHPExcel.php'),
